### PR TITLE
Defines github.com/ansible-security/ansible_collections.cisco.asa project

### PR DESCRIPTION
**Pull request by justjais**
Ansible Security Cisco ASA collection project

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -427,3 +427,11 @@
     name: github.com/pycontribs/selinux
     templates:
       - system-required
+
+- project:
+    name: https://github.com/ansible-security/ansible_collections.cisco.asa
```
Comment (pabelanger) [inline zuul.d/projects.yaml]: This should be github.com/ansible-collections/asa

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -427,3 +427,11 @@
...
+    templates:
+      - ansible-collections-cisco-asa
+      - ansible-python-jobs
+      - integrated-queue
+      - publish-to-galaxy-master-only
```
Comment (pabelanger) [inline zuul.d/projects.yaml]: For now, this should be:

templates:
  - system-required

File: zuul.d/projects.yaml [hunk 3]
```diff
@@ -427,3 +427,8 @@
     name: github.com/pycontribs/selinux
     templates:
       - system-required
+
+- project:
+    name: https://github.com/ansible-collections/asa
```
Comment (pabelanger) [inline zuul.d/projects.yaml]: drop https://

File: zuul/tenants.yaml [hunk 4]
```diff
@@ -29,6 +29,7 @@
           - ansible-collections/vmware
           - ansible-collections/vmware_rest
           - ansible-collections/vyos
+          - ansible-collections/asa
```
Comment (pabelanger) [inline zuul/tenants.yaml]: please sort this

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Pull request merged**